pkgs/build-support/dotnet/make-nuget-deps/default.nix

```diff
@@ -53,6 +69,8 @@ lib.makeOverridable(
           '';
 
           installPhase = ''
+            runHook preInstall
```

Comment:
This is a good idea, but we should really add `pre/postUnpack` and `postInstall` at the same time.  Can we put this in a separate PR for dotnet infra stuff?

Reply:
Moved to https://github.com/NixOS/nixpkgs/pull/337096